docs/en/observability/tutorials.asciidoc

```diff
@@ -31,10 +31,6 @@ Not sure which agent to use? Refer to
 
 * <<monitor-azure, Monitor Microsoft Azure with {beats}>>
 
-include::monitor-aws-agent.asciidoc[]
-
-include::monitor-aws-beats.asciidoc[]
```

Comment:
These files are still included, they just moved from `Tutorials` to `AWS monitoring`.

Reply:
The original files need to be deleted though. You've added new files instead of moving over the new files. For example, if I go to this PR in GitHub:

I can see the original files in `/docs/en/observability/`:
* https://github.com/alaudazzi/observability-docs/blob/aws-monitoring-doc-redesign/docs/en/observability/monitor-aws-agent.asciidoc
* https://github.com/alaudazzi/observability-docs/blob/aws-monitoring-doc-redesign/docs/en/observability/monitor-aws-beats.asciidoc

And I can see the new files in `/docs/en/observability/cloud-monitoring/`:
* https://github.com/alaudazzi/observability-docs/blob/aws-monitoring-doc-redesign/docs/en/observability/cloud-monitoring/aws/monitor-aws-agent.asciidoc
* https://github.com/alaudazzi/observability-docs/blob/aws-monitoring-doc-redesign/docs/en/observability/cloud-monitoring/aws/monitor-aws-beats.asciidoc

If you delete the original files in  `/docs/en/observability/`, the diff will correctly show history for the new files as it will then know that the file has moved.

Reply:
Got it. Removed the files in